app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    existing_file = command.split(" ")[1]
+    new_file = command.split(" ")[2]
+
+    if existing_file == new_file:
+        return
+
+    with (open(existing_file, "r") as f_read,
```

Comment:
You forgot to check if the command is `cp` (check `checklist` file in this module).